Assets/Resources/PrototypePrefab/PLayerZ.prefab

```diff
@@ -0,0 +1,563 @@
+%YAML 1.1
```

Comment:
If this file is a simple wrapper of the Player scripts.
I don't think they should name as these now.
You should name them according to their functionality.